src/rgw/rgw_rados.cc

```diff
@@ -3764,6 +3764,22 @@ int RGWRados::init_rados()
     }
   }
 
+  {
```

Comment:
we might like RGWRados to have more context--the type of instance is manifest at RGWStoreManager::get_storage(..) [rgw_main.cc, rgw_admin.cc, librgw.cc], and we'd like to be able to distinguish radosgw, nfs-ganesha instantiating librgw, and radosgw-admin, for sure